addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
More important as far as I can see this does not follow the archetype, but I still do not have glasses so I might be wrong 😉

https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding.test/src/main/resources/archetype-resources/pom.xml